pandas/core/common.py

```diff
@@ -539,6 +539,30 @@ class Sentinel(object):
     return Sentinel()
 
 
+def _get_range_parameters(data):
+    """Gets the start, stop, and step parameters from a range object"""
```

Comment:
maybe put this in ``pandas/compat/__init__``  instead

Reply:
done